### ISSUE TITLE
Docs include local development with new Spanner Emulator

### ISSUE THREAD

**Issue by longmuir** [akka/akka-persistence-spanner#88]
### Short description
Google Spanner now includes a local emulator that would be convenient for local development (https://cloud.google.com/spanner/docs/emulator). The getting started docs for Akka Persistence with Google Spanner should include some instructions on how to begin developing with Akka Persistence using this local emulator. 

### Details
Testing Akka Persistence applications locally can be challenging if you want to simulate an actual persistence backend (rather than just levelDB). This may provide a better way to validate using Akka Persistence with Spanner, and give developers more confidence.

**Comment by chbatey** [comment]
We have https://doc.akka.io/docs/akka-persistence-spanner/current/contributing.html

It is quite light on details, do you think we need any more?

**Comment by chbatey** [comment]
At least we should target it at end users and not just in the contributing section

**Comment by longmuir** [comment]
Yeah, we should have docs like that targeting end users - probably in Getting Started, providing a bit more guidance on setting up Akka Persistence with Spanner and running tests for development.
